app/redux/ducks/classify.spec.js

```diff
@@ -278,6 +307,26 @@ describe('Classifier actions', function () {
       const newState = reducer(state, action);
       expect(newState).to.deep.equal(state);
     });
+    it('should not record the lastInteventionUUID if not set', function () {
+      const newState = reducer(state, action);
+      expect(newState.classification.metadata.hasOwnProperty('intervention_uuid')).to.be.false;
+    });
+    describe('with lastInteventionUUID set', function () {
+      const interventionUUIDState = {
+        classification: { id: '1' },
+        workflow: { id: '1' },
+        upcomingSubjects: [mockSubject('1')],
+        lastInterventionUUID: '2d931510-d99f-494a-8c67-87feb05e1594'
+      };
+      it('should record the lastInteventionUUID as metadata.intervention_uuid', function () {
```

Comment:
Typo in the it statement. It should be `should record the lastInterventionUUID as metadata.intervention_uuid`

Reply:
```suggestion
      it('should record the lastInterventionUUID as metadata.intervention_uuid', function () {
```